Models

TeyimPila opened the pull request:
Closes #2 
- The attendance model

TeyimPila committed:
created models

TeyimPila committed:
created models

TeyimPila committed:
added minor configurations

changed manage.py to look for settings in settings/base.py and added a
SECRET_KEY variable in base.py

TeyimPila committed:
added minor configurations

changed manage.py to look for settings in settings/base.py and added a
SECRET_KEY variable in base.py

TeyimPila committed:
ignored secret variables

TeyimPila committed:
ignored secret variables

TeyimPila committed:
ignored secret files

TeyimPila committed:
ignored secret files

TeyimPila committed:
updated gitignore

TeyimPila committed:
updated gitignore

TeyimPila committed:
added some comments

TeyimPila committed:
added some comments

skaex commented on manage.py:
```diff
@@ -3,7 +3,8 @@
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ams.settings")
+    # os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ams.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ams.settings.base")
```
This is meant to be `ams.settings.development`

skaex commented on secret_variables.py:
```diff
@@ -0,0 +1 @@
+SECRET_KEY = "716i(1&-!co_s!n0j6o*(zrt05s-nlpbo4h9sr)v&%8&qmv^8m"
```
I told you to keep this out of

```
Source Control
```

skaex committed:
Percieved script JSON output (#6)

* added sample perceived json output sample for attendance extraction script

* Correct sample JSON

- Corrected the sample JSON file and placed it a new directory.
- Not happy with the sample data.

* Update to two students

- Update JSON to accommodate more students.

skaex committed:
Minor updates

- Update `.gitignore`
- Update management to dev settings not base.

skaex committed:
Remove secret

- Removed secret file from source control.

skaex committed:
Remove secret

- Removed secret file from source control.

skaex committed:
Merge branch 'develop' into feature/add-models

TeyimPila committed:
minor configs

TeyimPila committed:
minor configs

TeyimPila committed:
ignore

TeyimPila committed:
configured settings for the sake of future branches

skaex committed:
Merge pull request #7 from SkaeX/unify-settings

Unify Settings

TeyimPila committed:
completed rebase

SkaeX commented:
Approved
